src/custom/components/analytics/events/transactionEvents.ts

```diff
@@ -1,6 +1,14 @@
 import { OrderClass } from 'state/orders/actions'
 import { Category, sendEvent } from '../index'
 
+import { PIXEL_EVENTS } from '../pixel/constants'
+import { sendFacebookEvent } from '../pixel/sendFacebookEvent'
```

Comment:
ubernit: i think the file could be just facebook.ts for simplicity, also in case you want to expose other objects and not just sendFacebookEvent function

feel free to ignore, its also fine

Reply:
Good point 👍